clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2705,6 +2705,30 @@ An error will be given if:
   }];
 }
 
+def AMDGPUNumWorkGroupsDocs : Documentation {
+  let Category = DocCatAMDGPUAttributes;
+  let Content = [{
+The number of work groups specifies the number of work groups when the kernel
+is dispatched.
+
+Clang supports the
+``__attribute__((amdgpu_num_work_groups(<x>, <y>, <z>)))`` attribute for the
+AMDGPU target. This attribute may be attached to a kernel function definition
```

Comment:
'kernel function definition' probably needs more elaboration as well, we now have ~3 different 'kinds' of kernel function definitions.

Reply:
That would be preferential, yes

Reply:
Changed to what's suggested above.